packages/flutter/test/material/reorderable_list_test.dart

```diff
@@ -771,6 +798,33 @@ void main() {
         expect(findState(const Key('A')).checked, true);
       });
 
+      testWidgets('Preserves children states when rebuilt', (WidgetTester tester) async {
```

Comment:
Is this the same test as previous one?

Reply:
Yes. One is for''vertical mode" and the other is for "horizontal mode"

Reply:
I see, but then you will need to set scrollDirection to vertical in this test, right?

Reply:
That's right. I'll apply it right away. 👍